zetaclient/bitcoin/txscript.go

```diff
@@ -0,0 +1,227 @@
+package bitcoin
```

Comment:
Looks like we could add an underscore for filename more visual
```
tx_script.go
```

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/2bcdfbc727c12612159ffbf008bb00505a996b4b